provisioning/roles/chrome/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+- name: Download latest chrome
```

Comment:
Can't we use chromium instead? The advantage is that there are existing packages.